### ISSUE TITLE
Updating intl package to 0.19.0

### ISSUE THREAD

**Issue by Dreamersoul** [onepub-dev/money.dart#80]
is there any plans to update the intl package to 0.19.0?

**Comment by bsutton** [comment]
Yes.

I'm travelling at the moment but if you raise a pr I can publish it next
week.

**Comment by bsutton** [comment]
resolved in 4.1.0

**Comment by paricleu** [comment]
Why was this done? I'm on the flutter stable channel and use the official [l10n](https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization) which requires
```
dependencies:
  flutter:
    sdk: flutter
  flutter_localizations:
    sdk: flutter
```
Which has intl pinned to `0.18.1`:

> Note: intl is pinned to version 0.18.1 by flutter_localizations from the flutter SDK.

Which prevents me from using `money2: ^4.1.0`.
Are there any specific features from `intl: 0.19.0` this package needs or why can't it use `intl: ^0.18.1` which would allow @Dreamersoul to use `intl: 0.19.1` anyways if he wants to?

**Comment by Dreamersoul** [comment]
I believe that I was upgrading dependencies. And many of them are moving to 0.19

However I feel if you look at the change log you can use the previous version of money2 which uses intl 0.18
It won't impact your usage from what I'm seeing

**Comment by paricleu** [comment]
Yes, I know. But this blocks everyone on the stable channel that uses l10n, which are probably a lot of people (or not considering I'm the first to say something 😅), from using the next versions of this plugin and if 0.19.0 is not explicitly needed the version constraints should be the lowest possible (or at least reasonably old) to ensure all plugins work well together and `pub upgrade` can grab the newest possible version which works with all dependencies.
But I'm just nitpicking, have a nice day.

**Comment by bsutton** [comment]
so the update was at the request of another user that was having a conflict.

I guess one approach (in the short term) might be to run dual releases one with 0.18 and one with an 0.19 dep.

The problem with this idea is my bandwidth is rather limited. It would also require a parallel release of the fixed package (I'm the maintainer of fixed).

I think the first cab of the rank is getting 5.x released to stable.
It is ready to go I was just waiting for some feedback before releasing it.

If anyone could have a look at 5.0 and confirm that it looks sensible then I will get that released and then we can look at the 0.18 issue.

**Comment by bsutton** [comment]
I'm closing this. 5.0.0 is running on 0.19.0. It seems the easiest thing is for users to vender the package if the need 0.18 compatibility.
